libs/output-mapping/src/Storage/StoragePreparer.php

```diff
@@ -36,7 +35,7 @@ public function prepareStorageBucketAndTable(
         );
 
         if ($destinationTableInfo !== null) {
-            if ($processedSource->getSchema()) {
+            if ($tableChangesStore->hasChanges()) {
```

Comment:
Asi bych se nerozhodoval podle schema ale rovnou jestli nejaky changes jsou.
Je to trochu zamotany, ale $tableChangesStore se jen pokud schema je

https://github.com/keboola/platform-libraries/pull/284/files#diff-aa5a8dc48b81ba2bec17dfa52d0fdafcaaa00e2b9a9ae5686586a07bf95e39a9R115